WTWSMS/common/landed_titles/mercenaries.txt

```diff
@@ -2565,4 +2564,80 @@ d_ghanan_band = {
 	strength_growth_per_century = 1.0
```

Comment:
I'm putting this on hold for now: because there is too much I disagree on and a such overhaul was neither planned nor something I think is possible to do in this format.